.github/workflows/release.yml

```diff
@@ -0,0 +1,77 @@
+name: releaser
+
+on:
+  create:
+      tags:
+  # push:
+  #   branches:
+  #     - 'release'
+  #   tags:
+  #     - 'v*'
+
+jobs:
+  build:
+    name: Build release on ${{ matrix.os }} with ${{ matrix.compiler }}.
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            compiler: gcc
+            target: Linux
+
+          - os: macos-latest
+            compiler: clang
+            target: Macos
+
+          - os: windows-latest
+            compiler: cl
+            target: Windows
+    # if: github.event_name == 'create' && github.event.ref_type == 'tag' && startsWith(github.event.ref, 'v')
+    # https://stackoverflow.com/questions/59349905/which-properties-does-github-event-in-a-github-workflow-have
+    # https://github.community/t/how-to-get-just-the-tag-name/16241/15
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+      -
+        name: Cache Go modules
+        uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+            
+      # - name: Setup tmate session
```

Comment:
Better put a comment that why these lines are commented or remove these commented lines. ;)